storybook/src/components/Figure/Figure.stories.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { Image } from '@amsterdam/design-system-react'
+import { Figure } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+import { exampleCaption } from '../shared/exampleContent'
+
+const caption = exampleCaption()
+
+const meta = {
+  title: 'Components/Media/Figure',
+  component: Figure,
+} satisfies Meta<typeof Figure>
+
+export default meta
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const captionMeta = {
```

Comment:
I think we can just pass the `inverseColor` arg to the subcomponent in a `render` prop. That way, this control works for both stories.

Reply:
Yes, great!